style.css

```diff
@@ -351,5 +403,14 @@ h3.project-title {
 	.project-display {
 		border-width: 3px 0 0 0;
 	}
+
+	.contact-form {
+		min-width: 90%;
+	}
+
+	.input-email {;
```

Comment:
again typo?? or maybe it's some syntax I don't know about

Reply:
It's a typo, same as before, is repeated because I copied the selector to inside the media query.